sympy/functions/elementary/complexes.py

```diff
@@ -710,6 +792,22 @@ class conjugate(Function):
     2
     >>> conjugate(I)
     -I
+    >>> conjugate(3+2*I)
+    3 - 2*I
+    >>> conjugate(5-I)
```

Comment:
Put spaces around `+` and `-` in expressions, like `conjugate(3 + 2*I)` and `conjugate(5 - I)`.

Reply:
Done.